### PR TITLE
fix some comments

### PR DESCRIPTION
**Pull request by cuishuang**

### CODE REVIEW

**Review by ymesika — APPROVED** [review]
/ok-to-test

File: operator/pkg/apis/istio/v1alpha1/values_types.proto [hunk 1]
```diff
@@ -822,7 +822,7 @@ message TelemetryV2PrometheusConfig {
   ConfigOverride config_override = 3;
 }
 
-// Conrols telemetry v2 stackdriver settings.
+// Controls telemetry v2 stackdriver settings.
```
Comment (zirain) [inline operator/pkg/apis/istio/v1alpha1/values_types.proto:825]:
```suggestion
// TelemetryV2StackDriverConfig controls telemetry v2 stackdriver settings.
```

File: operator/pkg/apis/istio/v1alpha1/values_types.proto [hunk 2]
```diff
@@ -853,7 +853,7 @@ message TelemetryV2StackDriverConfig {
   AccessLogging inboundAccessLogging = 8;
 }
 
-// Conrols telemetry v2 access log policy filter settings.
+// Controls telemetry v2 access log policy filter settings.
```
Comment (zirain) [inline operator/pkg/apis/istio/v1alpha1/values_types.proto:856]:
```suggestion
// TelemetryV2AccessLogPolicyFilterConfig controls telemetry v2 access log policy filter settings.
```

**Comment by hzxuzhonghu** [comment]
@cuishuang I'd suggest making more meaningful contributions like bug fix, test coverage and feature requests, etc.

**Comment by cuishuang** [comment]
> 

Thanks, I am learning about the service mesh and reading part of the source code of istio. In the future, I will try to add unit testing and other related work for istio

**Comment by cuishuang** [comment]
> @cuishuang I'd suggest making more meaningful contributions like bug fix, test coverage and feature requests, etc.

Thanks, I am learning about the service mesh and reading part of the source code of istio. In the future, I will try to add unit testing and other related work for istio

**Review by hzxuzhonghu — COMMENTED** [review]
You will need to run make gen

**Review by howardjohn — APPROVED** [review]

**Comment by howardjohn** [comment]
> 

You can look for "good first issue" label on issues

**Comment by cuishuang** [comment]
> You will need to run make gen

done

**Comment by cuishuang** [comment]
> > 
> 
> You can look for "good first issue" label on issues

Thanks for the guidance

**Pull request merged**